CreeDictionary/CreeDictionary/templates/CreeDictionary/word-entries.html

```diff
@@ -31,7 +31,14 @@ <h2 class="definition-title definition-title--search-result">
                                         data-cy="linguistic-breakdown">{% if result.linguistic_breakdown_head %}
                                     {{ result.linguistic_breakdown_head|join:" + " }}
                                     + {% endif %}
-                                    <strong>{{ result.lemma_wordform.text }}</strong> + {{ result.linguistic_breakdown_tail|join:" + " }}</span>
+                                    <strong>{{ result.lemma_wordform.text|add:":" }}</strong>  <ol>                                    
+                                    {% for result in result.linguistic_breakdown_tail %}
+                                        <li>
+                                            {{ result }} +
```

Comment:
Remove the plus... that's just a leftover from the linguists :)

```suggestion
                                            {{ result }}
```